content/en/docs/reference/config/metrics/index.md

```diff
@@ -11,7 +11,7 @@ aliases:
 
 The following are the standard service level metrics exported by Istio.
 
-The telemetry component is [implemented](https://github.com/istio/proxy/tree/master/extensions/stats) as a [Proxy-wasm](https://github.com/proxy-wasm/spec) plugin.
+The telemetry component is implemented as a [Proxy-wasm](https://github.com/proxy-wasm/spec) plugin.
```

Comment:
this's not correct now, stats is a [native(c++) filter](https://github.com/istio/proxy/tree/master/source/extensions/filters/http/istio_stats).
cc @kyessenov

Reply:
Yeah, I'd say that components are implemented as "proxy extensions" without going into detail. MX is still Wasm, but it's really not important to a user of Istio, as long as it works.

Reply:
Do you suggest I change line 14 to `The telemetry component is implemented as a proxy extension` ? @kyessenov

Reply:
Hello, please let me know if there are any changes required, if any. If not I will probably mark this PR as closed if it's not needed.

Reply:
yes, that would be better.

Reply:
Hmm I'm trying to figure out what's going on with my EasyCLA auth, please bear with me.

Reply:
All good with the EasyCLA :) sorry about that!